File: Unity Project/SuperEngineer/Assets/Scripts/gameMaster.js
#pragma strict

static var currentScore : int = 0;
var test : int;
var style : GUIStyle = null;

function Update(){
	test = currentScore;
}

function OnGUI()
{
	GUI.color = Color.black;
	GUI.Label(Rect(Screen.height*1.1, Screen.width*.05,100 , 50),"Score: "+currentScore, style);
}

File: Unity Project/SuperEngineer/Assets/Scripts/GameMaster.cs
using UnityEngine;
using System.Collections;

public class GameMaster : MonoBehaviour 
{
	public static int points;

	void Start ()
	{
		points = 0;
	}

	void OnGUI ()
	{
		GUI.skin.textArea.alignment = TextAnchor.MiddleCenter;
		GUI.Label (new Rect (0, 0, Screen.width, 40), "Score: " + points);
	}
}
